packages/apps/tools/src/pages/transactions/module-explorer/utils.ts

```diff
@@ -25,8 +25,8 @@ export const getQueryValue = (
 export const getCookieValue = (
   needle: string,
   haystack: NextApiRequestCookies,
-  defaultValue?: string,
-): string | null => {
+  defaultValue?: string | [any],
+): string | [any] | [] | null => {
```

Comment:
Btw what is the expected allowed return type here? An array (tuple) with one item of type `any`, or should it be `Array<any>` (or `any[]`).

Sidenote; I would just like for TS to infer these types, so ESLint giving us an error for not explicitly declaring the return type. What do you think?

Reply:
Do you think it would be better to add the type here Array<INetworkData> so we avoid the TS errors? We can add this as a possible return type

Reply:
You know what might be better; since the defaultValue could be `any`thing, maybe it's better to make it a generic and let TS infer the return type itself. What do you think?

Reply:
I agree and it will better be a generic

Reply:
FYI; made it generic 👌